assets/js/job-editor/JobEditorMain.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import JobEditor from "./JobEditor"
+import type { WithActionProps } from "../react/lib/with-props";
+import { sortMetadata } from "../metadata-loader/metadata";
+import { useWorkflowStore } from "../workflow-editor/store";
+import pDebounce from "p-debounce";
+import { EDITOR_DEBOUNCE_MS } from "../common";
+import type { Lightning } from "#/workflow-diagram/types";
+
+interface JobEditorComponentProps {
+  job_id: string,
+  adaptor: string,
+  source: string,
```

Comment:
This is like `initialSource`, right? This is the source that will be shown until the store is connected. Do you think we should rename the prop or just leave it for now?

Reply:
Exactly. renaming the prop will also require renaming the attr name in phoenix. I'm okay with either tho